components/function-controller/internal/controllers/serverless/build_resources.go

```diff
@@ -88,7 +88,7 @@ func (r *FunctionReconciler) buildJob(instance *serverlessv1alpha1.Function, con
 						{
 							Name:  "executor",
 							Image: r.config.Build.ExecutorImage,
-							Args:  []string{fmt.Sprintf("--destination=%s", imageName), "--insecure", "--skip-tls-verify"},
+							Args:  append(r.config.Build.ExecutorArgs, fmt.Sprintf("--destination=%s", imageName), "--context=dir:///workspace"),
```

Comment:
`--destination` and `--context` should be configurable as it is strictly related to volumes and docker repository...